src/SessionMiddleware.php

```diff
@@ -61,10 +61,11 @@ public function handleRequest(Request $request, RequestHandler $requestHandler):
             $response->setCookie(new ResponseCookie($this->cookieName, $id, $this->cookieAttributes));
         }
 
-        $cacheControl = Http\parseFieldValueComponents($response, 'cache-control') ?? [];
+        $cacheControl = Http\parseMultipleHeaderFields($response, 'cache-control') ?? [];
 
         $tokens = [];
         foreach ($cacheControl as [$key, $value]) {
+            \assert($key !== null);
```

Comment:
Based on the type hint defined I wouldn't expect psalm to see the `$key` as null. However, static analysis complained about this. After having looked at the function that generates this key it seems like this will always be a `non-empty-string` in practice.